src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs

```diff
@@ -133,13 +135,13 @@ public async void HandleAsync_GivenRequestWithSearchRef_EvidenceDataAccessShould
         {
             //arrange
             var searchRef = TestFixture.Create<string>();
-            var request = GetAatfNotesRequest(SystemTime.UtcNow.Year, searchRef);
+            var wasteTypeList = TestFixture.CreateMany<WasteType>(2).ToList();
```

Comment:
Magic string (well int but yno) here, better to use the enum?

Reply:
Changed code as suggested